conn-fix/README.md

```diff
@@ -0,0 +1,359 @@
+defaultApplVerID: 9
+
+# Determines if sequence numbers should be reset to 1 after a normal logout termination
+resetOnLogout: false
+
+# Determines if sequence numbers should be reset to 1 after an abnormal termination
+resetOnDisconnect: false
+
+# Save Heartbeats to logs or not.
+logHeartbeats: false
+
+# Check required tags or not.
+checkRequiredTags: true
+
+# If set to checked, messages must be received from the counterparty within a defined number of seconds (see MaxLatency).
+# It is useful to turn this off if a system uses localtime for it's timestamps instead of GMT.
+checkLatency: false
+
+# If CheckLatency is set to checked, this defines the number of seconds latency allowed for a message to be processed. Default is 120.
+maxLatency: 120
+
+# If set to checked, allow unknown fields in messages. This is intended for unknown fields with tags < 5000 (not user defined fields).
+allowUnknownMsgFields: false
+
+# Determines if milliseconds should be added to timestamp fields. Only available for AML 3
+millisecondsInTimeStampFields: true
+
+# Determines if microseconds should be added to timestamp fields. Only available for AML 3. This value is more priority than 'Milliseconds In Time Stamp Fields'
+microsecondsInTimeStampFields: false
+
+# Receive limit in bytes to emulate Slow Consumer
+receiveLimit: 0
+
+# Client send reject if message is invalid
+rejectInvalidMessage: true          
+
+# If set to unchecked, user defined fields will not be rejected if they are not defined in the data dictionary, or are present in messages they do not belong to.
+validateUserDefinedFields: true
+
+# If set to unchecked, fields that are out of order (i.e. body fields in the header, or header fields in the body) will not be rejected. Useful for connecting to systems which do not properly order fields.
+validateFieldsOutOfOrder: true
+
+# If set to unchecked, messages which contain values not presented in the range will not be rejected.
+validateFieldsOutOfRange: true
+
+# If set to unchecked, fields without values (empty) will not be rejected. Useful for connecting to systems which improperly send empty tags.
+validateFieldsHaveValues: true
+
+# If set to checked, no reject sent on incoming message with duplicate tags Allow duplicated tags.
+duplicateTagsAllowed: false
+
+# If specified, the session start and end will be converted from default zone to UTC
+useLocalTime: false
+
+# Enables SSL usage for QFJ acceptor or initiator
+useSSL: false
+
+# KeyStore to use with SSL
+sslKeyStore: ""
+
+# KeyStore password to use with SSL
+sslKeyStorePassword: ""
+
+# Controls which particular protocols for secure connection are enabled for handshake
+sslEnabledProtocols: ""
+
+# Controls which particular SSL cipher suites are enabled for secure connection
+sslCipherSuites: ""
+
+# Enables fields ordering in raw message by dictionary
+orderingFields: false
+
+# Check the next expected target SeqNum against the received SeqNum. Default is checked.
```

Comment:
**Checks** the next expected target SeqNum against the received SeqNum. **Checked by default.**